packages/vscode-extension/media/step_commands.svg

```diff
@@ -1,9 +1,58 @@
-<svg width="426" height="320" viewBox="0 0 426 320" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-<rect width="426" height="320" fill="url(#pattern0)"/>
+<svg width="412" height="342" viewBox="0 0 412 342" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1209_105830)">
+<rect width="412" height="342" rx="4" fill="#1E1E1E"/>
+<path d="M0 4C0 1.79086 1.79086 0 4 0H408C410.209 0 412 1.79086 412 4V31H0V4Z" fill="#292929"/>
```

Comment:
Does the svgs apply to different color themes?

Reply:
no. I will commit other PR to support the different color themes.